composer.json

```diff
@@ -40,7 +40,9 @@
     "config": {
         "allow-plugins": {
             "bamarni/composer-bin-plugin": true,
-            "laminas/laminas-dependency-plugin": true
+            "laminas/laminas-dependency-plugin": true,
+            "elendev/composer-push": true,
+            "phpstan/extension-installer": true
```

Comment:
Is this diff expected? We don't use these added plugins anymore.

Reply:
composer install or update added these automatically. Ill remove.